fearless/Modules/Send/Validators/SendDataValidatingFactory.swift

```diff
@@ -299,4 +317,29 @@ class SendDataValidatingFactory: NSObject {
             }
         }
     }
+
+    private func minAssetAmount(
+        originCHainId: ChainModel.Id,
+        destChainId: ChainModel.Id
+    ) -> String {
+        let originKnownChain = Chain(chainId: originCHainId)
+        let destKnownChain = Chain(chainId: destChainId)
+
+        switch (originKnownChain, destKnownChain) {
```

Comment:
what if some of those chains (polkadot, kusama, acala, liberland) will add 'XOR' asset and you call this method to determine minimum amount of xor to transfer from polkadot to sora ?